rollup.config.js

```diff
@@ -84,7 +84,7 @@ export const getRollupConfig = ({ pwd, babelConfigFile }) => {
   const esConfig = {
     input,
     output: {
-      dir: `${SOURCE_DIR}/${pkgConfig?.module?.replace('/index.js', '')}`,
+      dir: `${SOURCE_DIR}/${pkgConfig.module.replace('/index.js', '')}`,
```

Comment:
@lordpixel was there a reason we needed to remove the optional chaining in Node 12 here? I think this syntax was used for packages that might not necessarily have a `module` parameter declared.

Reply:
The elvis op. sees not to be supported. Rollup didn't like it.